pkg/api/v1/helper/helpers.go

```diff
@@ -524,3 +524,28 @@ func StorageNodeAffinityToAlphaAnnotation(annotations map[string]string, affinit
 	annotations[v1.AlphaStorageNodeAffinityAnnotation] = string(json)
 	return nil
 }
+
+// NodeConditionEqual returns true if all effective types ("Status" is true) equals;
```

Comment:
hm, I think this's "equal" for the two node condition; if we want an exactly equal, we can use DeepEqual for all fields, including timestamps.